core/controllers/reader_test.py

```diff
@@ -1488,50 +1447,84 @@ def test_playthrough_not_stored_at_limiting_value(self):
             '/explorehandler/store_playthrough/%s' % (self.exp_id),
             {
                 'playthrough_data': self.playthrough_data,
-                'exp_issue_dict': self.exp_issue_dict
+                'issue_schema_version': 1
             }, self.csrf_token)
         self.process_and_flush_pending_tasks()
 
         model = stats_models.ExplorationIssuesModel.get_model(self.exp_id, 1)
         self.assertEqual(len(model.unresolved_issues), 1)
         self.assertEqual(len(model.unresolved_issues[0]['playthrough_ids']), 5)
 
-    def test_error_on_invalid_exp_issue_dict(self):
-        """Test that passing an invalid exploration issue dict raises an
+    def test_error_without_schema_version_in_payload_dict(self):
+        """Test that passing a payload without schema version raises an
         exception.
         """
-        del self.exp_issue_dict['issue_type']
-
         self.post_json(
```

Comment:
@brianrodri Thats not the case anymore. When I was working on this, I found out that passing in an exploration issue dict is redundant since all the information is contained in the playthrough data dict. So, im not passing the exploration issue dict at all, and passing an explicit issue_schema_version in the payload. So, in this test, there is no schema_version and only the playthrough_data in the payload.

Reply:
Ah, I'm still talking about the new code (GitHub put the comment on this side for some reason).

I'm saying to use a variable in the test-code to describe why the payload is incorrect:

```python
payload_dict_without_schema_version = {
    'playthrough_data': self.playthrough_data
}

self.post_json(..., payload_dict_without_schema_version, ...)

Reply:
Done.